services/ui-src/src/utils/validation/schemas.ts

```diff
@@ -17,30 +17,28 @@ export const textOptional = () => text().notRequired();
 // NUMBER - Helpers
 const validNAValues = ["N/A", "Data not available"];
 
-const ignoreCharsForSchema = (value: string, charsToReplace: RegExp) => {
+const valueCleaningNumberSchema = (value: string, charsToReplace: RegExp) => {
   return numberSchema().transform((_value) => {
     return Number(value.replace(charsToReplace, ""));
   });
 };
 
-// NUMBER - Number or Valid Strings
 export const number = () =>
-  mixed()
-    .test({
-      message: error.REQUIRED_GENERIC,
-      test: (val) => val != "",
-    })
+  string()
     .required(error.REQUIRED_GENERIC)
     .test({
       message: error.INVALID_NUMBER_OR_NA,
-      test: (val) => {
-        const replaceCharsRegex = /[,.]/g;
-        return (
-          ignoreCharsForSchema(val, replaceCharsRegex).isValidSync(val) ||
-          validNAValues.includes(val)
-        );
+      test: (value) => {
+        const validNumberRegex = /[0-9,.]/;
+        if (value) {
+          const isValidStringValue = !!validNAValues.includes(value);
+          const isValidNumberValue = !!value?.match(validNumberRegex);
+          return isValidStringValue || isValidNumberValue;
+        }
+        return true;
```

Comment:
Nit: This should really be `else return true`.
```suggestion
        } else return true;
```